Index.html

```diff
@@ -13,15 +13,28 @@
 </head>
 <body>
     <header>
-        <span><a href="#"><b>My Logo</b></a></span>
-        <nav id="switch">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
+      <a href="#" id="Logo"><b>David Katende</b>
+        <nav  class="nav-container">
+          <ul class="nav-menu">
+            <li>    
+              <a href="#portfolio">Portfolio</a>    
+            </li>    
+            <li >    
+              <a href="#about" >About</a>    
+            </li>    
+            <li >    
+              <a href="#contact" >Contact</a>    
+            </li>    
+          </ul>
+            <div id="switch">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>            
         </nav>
     </header>
     <section class="intro">
-        <h1 >I’m David. Glad to see you!</h1>
+        <h1>I’m Alexandra. Glad to see you!</h1>
         <p>I’m a software developer! I can help you build a product , feature or website Look through
             some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate
             to contact me.</p>
```

Comment:
Good job implementing the changes 👍 As suggested by the reviewer [here](https://github.com/Katended/mobilefirstproject2/pull/4#discussion_r1159707025), please make sure to align your text with the one we have in the design. That means to do the following 👇
- [ ] Kindly make the header text to appear in two lines. That means _"Glad to see you!"_ should be on 2nd line as given in Figma guidelines. You can use a [line break element](https://developer.mozilla.org/en-US/docs/web/html/element/br) for that ✨
- [ ] Kindly make sure to reduce the width of `p` tag so that it appears on the screen as given in the Figma template as suggested by the previous reviewer [here](https://github.com/Katended/mobilefirstproject2/pull/4#discussion_r1159707025). This you will show the confidence in your ability to replicate any given design which you will be doing in a job market as well.

Your design 🔽 | Figma guidelines 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/230495286-6368b8bb-24eb-496d-b8ce-cf6910648cbd.png) | ![image](https://user-images.githubusercontent.com/106119105/230495349-7ea844e9-8b20-4e91-9e50-a11969cc1512.png) |